Dockerfile

```diff
@@ -1,16 +1,21 @@
 FROM alpine:3.13
 
 ENV TERRAFORM_VERSION=0.13.6
+ARG TARGETPLATFORM
 
 RUN apk add --no-cache \
         bash \
         curl \
         jq \
  && apk upgrade \
- && curl -o terraform.zip https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip \
- && unzip terraform.zip \
+ && if [ "${TARGETPLATFORM}" = "linux/amd64" ]; then TERRAFORM_PLATFORM="linux_amd64"; fi; \
+    if [ "${TARGETPLATFORM}" = "linux/arm/v7" ]; then TERRAFORM_PLATFORM="linux_arm"; fi; \
+    if [ "${TARGETPLATFORM}" = "linux/arm64" ]; then TERRAFORM_PLATFORM="linux_arm64"; fi; \
+    if [ "${TERRAFORM_PLATFORM}" = "" ]; then TERRAFORM_PLATFORM="${TARGETPLATFORM}"; fi \
+ && curl -o terraform_${TERRAFORM_PLATFORM}.zip https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_${TERRAFORM_PLATFORM}.zip \
+ && unzip terraform_${TERRAFORM_PLATFORM}.zip \
  && mv terraform /usr/local/bin/ \
- && rm terraform.zip \
+ && rm terraform_${TERRAFORM_PLATFORM}.zip \
```

Comment:
I suggest to keep it as `terraform.zip`. Otherwise good to go

Reply:
Done